src/components/attendee/scss/Gradients.module.scss

```diff
@@ -8,4 +7,13 @@
     repeating-linear-gradient(90deg, #060606, #000 11px),
     repeating-linear-gradient(30deg, #060606, #000 13px);
   background-blend-mode: screen;
+
```

Comment:
[NOTE] These look like styles that might work as a new module under `./src/scss/attendee/effects/`